### PR TITLE
Adding test for all beans

### PR DESCRIPTION
**Pull request by chloedaull**
for now only Order is done.

### CODE REVIEW

File: test/com/shashi/beans/OrderBeanTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        String transactionId = "T123";
+        String productId = "P456";
+        int quantity = 5;
+        double amount = 100.0;
+
+        OrderBean orderBean = new OrderBean(transactionId, productId, quantity, amount);
+
+        assertEquals(transactionId, orderBean.getTransactionId());
+        assertEquals(productId, orderBean.getProductId());
+        assertEquals(quantity, orderBean.getQuantity());
```
Comment (Transurgeon) [inline test/com/shashi/beans/OrderBeanTest.java:59]: this is wrong, please change it

Comment (cedparadis) [reply test/com/shashi/beans/OrderBeanTest.java:59]: You can do tests on beans if you want but I dont think its that important to do tests for beans as they do not have methods except getter and setter

Comment (cedparadis) [reply test/com/shashi/beans/OrderBeanTest.java:59]: Focus should be more on service implementation classes

**Comment by chloedaull** [comment]
this is outdated

**Pull request closed without merging**